erpnext/accounts/report/general_ledger/general_ledger.py

```diff
@@ -478,7 +486,19 @@ def get_columns(filters):
 			"options": "Project",
 			"fieldname": "project",
 			"width": 100
-		},
+		}
+	])
+
+	if filters.get("include_dims"):
```

Comment:
```suggestion
	if filters.get("include_dimensions"):
```

Reply:
OK, I think that covers all of your comments. I will test again and force-push the revised PR shortly. Thanks for your feedback! -- Glen